pkg/kubernetes/scripts/policy.js

```diff
@@ -190,7 +190,69 @@
 
                 return methods.create([binding], namespace);
             }
+            function removeRoleFromPolicyBinding(policyBinding, project, role, subject) {
+                var roleBinding, i, defaultPolicybinding;
+                var roleBindings = [];
+                var namespace = toName(project);
+
+                if(policyBinding && policyBinding.one()){
+                    defaultPolicybinding = policyBinding.one();
+                    roleBindings = defaultPolicybinding.roleBindings;
+                }
+                var patchData = { "roleBindings": roleBindings };
+                for (i = roleBindings.length - 1; i >= 0; i--) {
+                    if(roleBindings[i].name === role) {
+                        roleBinding = roleBindings[i].roleBinding;
+                        break;
+                    }
+                }
+                removeFromArray(roleArray(roleBinding, "subjects"), subject);
+                removeFromArray(roleArrayKind(roleBinding, subject.kind), subject.name);
+                return methods.patch(defaultPolicybinding, patchData);  
+            }
+            function addRoleToPolicyBinding(policyBinding, project, role, subject) {
```

Comment:
Isn't this function a duplicate of addToRole below. I don't think it's necessary if so.